plugins/csi/client.go

```diff
@@ -254,6 +254,27 @@ func (c *client) ControllerPublishVolume(ctx context.Context, req *ControllerPub
 	}, nil
 }
 
+func (c *client) ControllerUnpublishVolume(ctx context.Context, req *ControllerUnpublishVolumeRequest) (*ControllerUnpublishVolumeResponse, error) {
+	if c == nil {
+		return nil, fmt.Errorf("Client not initialized")
+	}
+	if c.controllerClient == nil {
+		return nil, fmt.Errorf("controllerClient not initialized")
+	}
+
+	upbrequest := &csipbv1.ControllerUnpublishVolumeRequest{
+		VolumeId: req.VolumeID,
+		NodeId:   req.NodeID,
```

Comment:
We should validate that this is not empty - CSI will unpublish the volume from _all_ nodes if this is empty https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerunpublishvolume

Reply:
Yikes! Will do!

Reply:
Fixed and added tests to match.